### PR TITLE
add clusterpedia openapi and bump k8s dependency to v1.28.2

### PR DESCRIPTION
**Pull request by wawa0210**
**What type of PR is this?**

/kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

The reason for adding ignored static check SA1019 is that a method is marked as deprecated, but the changes are relatively large and have not been changed yet. Later we can try to modify PR alone .The detailed error is as follows

```bash
  Running [/home/runner/golangci-lint-1.53.3-linux-amd64/golangci-lint run --out-format=github-actions] in [] ...
  Error: SA1019: wait.NewExponentialBackoffManager is deprecated: Will be removed when the legacy Poll methods are removed. Callers should construct a Backoff struct, use DelayWithReset() to get a DelayFunc that periodically resets itself, and then invoke Timer() when calling wait.BackoffUntil. (staticcheck)
  Error: SA1019: wait.NewExponentialBackoffManager is deprecated: Will be removed when the legacy Poll methods are removed. Callers should construct a Backoff struct, use DelayWithReset() to get a DelayFunc that periodically resets itself, and then invoke Timer() when calling wait.BackoffUntil. (staticcheck)
  
  level=warning msg="[runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
  level=warning msg="[runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
  level=warning msg="[runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
  
  Error: issues found

```
detail link --> [href](https://github.com/clusterpedia-io/clusterpedia/actions/runs/6648351342/job/18065247015?pr=579)

relation code 

*   https://github.com/clusterpedia-io/clusterpedia/blob/e788be497328365971b362f5ec41011b452e9132/pkg/synchromanager/clustersynchro/informer/reflector.go#L167
* https://github.com/clusterpedia-io/clusterpedia/blob/e788be497328365971b362f5ec41011b452e9132/pkg/synchromanager/clustersynchro/informer/cache/.reflector.go#L182

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -32,5 +32,9 @@ linters-settings:
       - Standard
       - Default
       - Prefix(github.com/clusterpedia-io)
+  staticcheck:
+    checks:
+      - all
+      - "-SA1019" # disable the rule SA1019(Using a deprecated function, variable, constant or field)
```
Comment (Iceber) [inline .golangci.yml:38]: Is this necessarily related to this feature?
For version upgrades, I think it's important to ensure as few changes as possible, and this looks like it could create a new pr

File: staging/src/github.com/clusterpedia-io/api/github.com/wawa0210/clusterpedia/staging/src/github.com/clusterpedia-io/api/cluster/v1alpha2/zz_generated.deepcopy.go [hunk 2]
```diff
@@ -0,0 +1,322 @@
+//go:build !ignore_autogenerated
```
Comment (Iceber) [inline staging/src/github.com/clusterpedia-io/api/github.com/wawa0210/clusterpedia/staging/src/github.com/clusterpedia-io/api/cluster/v1alpha2/zz_generated.deepcopy.go:1]: A personal path `github.com/wawa0210/clusterpedia/...`

**Comment by Iceber** [comment]
> bump k8s dependency to latest version

It is recommended that the commit message and pr title be changed to be specific version
> bump k8s dependency to v1.28.2

File: staging/src/github.com/clusterpedia-io/api/github.com/wawa0210/clusterpedia/staging/src/github.com/clusterpedia-io/api/cluster/v1alpha2/zz_generated.deepcopy.go (hunk 2 above)
Comment (wawa0210) [reply staging/src/github.com/clusterpedia-io/api/github.com/wawa0210/clusterpedia/staging/src/github.com/clusterpedia-io/api/cluster/v1alpha2/zz_generated.deepcopy.go]: done

File: .golangci.yml (hunk 1 above)
Comment (wawa0210) [reply .golangci.yml:38]: The reason has been added, thanks for the reminder
